tests/Http/Controllers/ReviewTest.php

```diff
@@ -0,0 +1,424 @@
+            'content' =>
+                'Officia perspiciatis est quam aperiam. Dolor voluptatibus nemo vel molestiae quia aut. Dolor reprehenderit autem dolor eaque.
+                Molestiae blanditiis veniam quis iure officiis explicabo. Rerum veniam et nam voluptatem itaque quaerat omnis.
+                Necessitatibus voluptatem odit eaque repudiandae voluptatem qui. Ea et alias maiores sint aliquam qui veniam eaque.
+                Saepe occaecati id aut doloremque repellat. Maiores neque deserunt dolores numquam quia ab quam.'
+        ]);
+    }
+
+    /** @test */
+    public function a_user_can_update_their_review()
+    {
+        $user = User::factory()->create();
+        $movie = Movie::factory()->create();
+        $review = Review::factory()->create([
+            'user_id' => $user->id,
+            'movie_id' => $movie->id,
+            'title' => 'Review Title Sample',
+            'rating' => 4.8,
+            'content' => 'Sample review content',
+        ]);
+
+        $this->actingAs($user)
+            ->putJson("/reviews/{$review->id}", [
+                'id' => $review->id,
+                'user_id' => $user->id,
+                'movie_id' => $movie->id,
+                'title' => 'Mock Turtle, suddenly.',
+                'rating' => 4.8,
+                'content' =>
+                    'Officia perspiciatis est quam aperiam. Dolor voluptatibus nemo vel molestiae quia aut. Dolor reprehenderit autem dolor eaque.
+                Molestiae blanditiis veniam quis iure officiis explicabo. Rerum veniam et nam voluptatem itaque quaerat omnis.
+                Necessitatibus voluptatem odit eaque repudiandae voluptatem qui. Ea et alias maiores sint aliquam qui veniam eaque.
+                Saepe occaecati id aut doloremque repellat. Maiores neque deserunt dolores numquam quia ab quam.'
+            ])
+            ->assertRedirect("/reviews/{$review->id}")
+            ->assertSessionHas('status', 'Success! Review has been updated.');
+
+        $this->assertDatabaseHas('reviews', [
+            'id' => $review->id,
+            'user_id' => $user->id,
+            'movie_id' => $movie->id,
+            'title' => 'Mock Turtle, suddenly.',
+            'rating' => 4.8,
+            'content' =>
+                'Officia perspiciatis est quam aperiam. Dolor voluptatibus nemo vel molestiae quia aut. Dolor reprehenderit autem dolor eaque.
+                Molestiae blanditiis veniam quis iure officiis explicabo. Rerum veniam et nam voluptatem itaque quaerat omnis.
+                Necessitatibus voluptatem odit eaque repudiandae voluptatem qui. Ea et alias maiores sint aliquam qui veniam eaque.
+                Saepe occaecati id aut doloremque repellat. Maiores neque deserunt dolores numquam quia ab quam.'
+        ]);
+    }
+
+    /** @test */
+    public function any_auth_user_cannot_edit_a_review_they_did_not_create()
+    {
+        $user = User::factory()->create([
+            'id' => 1
```

Comment:
You can just use the create Jayd :D no need to specify the id here because we ate checking the id by $user->id